libs/ui-lib-tests/cypress/integration/static-ip/3-configure-static-ip-yaml.cy.ts

```diff
@@ -57,6 +57,14 @@ describe(`Assisted Installer Static IP YAML configuration`, () => {
       commonActions.getDangerAlert().should('be.visible');
       commonActions.verifyNextIsDisabled();
     });
+
+    it('Can configure static IP in YAML view uploading a file directly', () => {
+      staticIpPage.yamlView.fileUpload().attachFile(`static-ip/files/test.yaml`);
+      staticIpPage.yamlView.macAddress().type('00:00:5e:00:53:af');
+      staticIpPage.yamlView.interface().type('interface1');
+
+      commonActions.getNextButton().should('be.enabled');
```

Comment:
The `getNextButton` function is no longer exposed, as it made the tests more frail.

For checking the button validation, we need to use `verifyNextIsDisabled / verifyNextIsEnabled`